backend/app/routes/index.js

```diff
@@ -5,11 +5,15 @@ const { emailTest } = require('./testRoutes/emailTest');
 const tinyURL = require('./tinyURL');
 const broadcast = require('./broadcast');
 const FAQ = require('./faq/post');
+const GetFaq = require('./faq/GetFaq');
```

Comment:
please use camelCase in variable and usenames for eg thi sline can be replaced with - 
`const getFaq = require('./faq/get-faq');`